asv_bench/benchmarks/utils/common.py

```diff
@@ -455,23 +455,30 @@ def trigger_import(*dfs):
     ----------
     *dfs : iterable
         DataFrames to trigger import.
+
+    Returns
+    -------
+    tuple of DataFrames
+        Returns exactly the same objects that were passed into the function.
```

Comment:
adding return here to ease variables initialization by already imported frames (see [here](https://github.com/dchigarev/modin/blob/817ca9b1d5de9a538346880ce9c26d68457daa47/asv_bench/benchmarks/benchmarks.py#L518))